modules/base/src/processors/transform.cpp

```diff
@@ -160,4 +160,42 @@ mat4 CustomTransformProperty::getMatrix() const { return matrix; }
 
 }  // namespace transform
 
+TransformListProperty::TransformListProperty(const std::string& identifier,
+                                             const std::string& displayName,
+                                             InvalidationLevel invalidationLevel,
+                                             PropertySemantics semantics)
+    : CompositeProperty(identifier, displayName, invalidationLevel, semantics)
+    , transforms_(
+          "transforms", "Transformations",
+          []() {
+              std::vector<std::unique_ptr<Property>> v;
+              v.emplace_back(
+                  std::make_unique<transform::TranslateProperty>("translation", "Translation"));
+              v.emplace_back(std::make_unique<transform::RotateProperty>("rotation", "Rotation"));
+              v.emplace_back(std::make_unique<transform::ScaleProperty>("scaling", "Scaling"));
+              v.emplace_back(
+                  std::make_unique<transform::CustomTransformProperty>("custom", "Custom Matrix"));
+              return v;
+          }())
+    , result_("result", "Result", mat4(1.0f),
```

Comment:
This property should not be possible to change from the outside (or linked). 
Add constraints

Reply:
We can not make a property unlinkable

Reply:
Should be ConstraintBehavior::Ignore?

Reply:
yes, but that does not affect linking of the value. Only the boundary